src/framework/node.ml

```diff
@@ -35,6 +37,12 @@ let pretty_trace () = function
   | Function      fd -> dprintf "call of %s" fd.svar.vname
   | FunctionEntry fd -> dprintf "entry state of %s" fd.svar.vname
 
+(** Output functions for Printable interface *)
+let pretty () x = pretty_trace () x
```

Comment:
It's a bit debatable here, which of the existing `pretty_*` variants should be the default. But since we are using a mix of them all over the place for different information and there's no true preferred way, then I guess this is good enough for now. Since there was no `pretty` here, nothing else depended or will get changed by this new default.